src/include/threadpool/mono_queue_pool.h

```diff
@@ -11,45 +11,48 @@
 //===----------------------------------------------------------------------===//
 
 #pragma once
-#include "task_queue.h"
-#include "worker_pool.h"
 
-// TODO: tune these variables
-#define DEFAULT_WORKER_POOL_SIZE 4
-#define DEFAULT_TASK_QUEUE_SIZE 32
+#include "worker_pool.h"
 
 namespace peloton {
 namespace threadpool {
+
+// TODO: tune these variables
```

Comment:
When you merge the following multithreaded PR, let's put these into configuration(settings).   This is not required to change in this PR.